modules/ROOT/examples/live-demos/merge-tag/index.html

```diff
@@ -0,0 +1,8 @@
+<textarea id="mergetags">
+    <h1>Proposal for Services — {*Client.Name*}</h1>
```

Comment:
Indentation is incorrect in this file. It should be using 2 spaces not 4.

- [ ] Fix indentation

Reply:
Indents halved.